gc/base/standard/ConfigurationGenerational.cpp

```diff
@@ -219,4 +219,20 @@ MM_ConfigurationGenerational::defaultMemorySpaceAllocated(MM_GCExtensionsBase *e
 	extensions->setGuaranteedNurseryRange(nurseryStart, (void*)UDATA_MAX);
 }
 
+uintptr_t
+MM_ConfigurationGenerational::calculateDefaultRegionSize(MM_EnvironmentBase *env)
+{
+	uintptr_t regionSize = STANDARD_REGION_SIZE_BYTES;
+
+#if defined(OMR_GC_CONCURRENT_SCAVENGER)

```

Comment:
There was a new API added by @amicic.  Can you switch to using that API for code like this?  Thanks